src/peft/utils/integrations.py

```diff
@@ -101,6 +104,7 @@ def dequantize_bnb_weight(weight: torch.nn.Parameter, state=None):
             weight = weight.to(torch.device("cuda"))
         elif is_xpu_available():
             weight = weight.to(torch.device("xpu"))
+            state.SCB = state.SCB.to(torch.device("xpu"))
```

Comment:
`weight` is moved to accelerator, but `SCB` not, will make later `int8_vectorwise_dequant` cry to say `weight in xpu, but SCB in cpu` and make CPU CI, fail, so move it to accelerator too.